client_test.go

```diff
@@ -175,12 +182,66 @@ func TestClientSyncBadID(t *testing.T) {
 	m := new(Msg)
 	m.SetQuestion("miek.nl.", TypeSOA)
 
+	c := &Client{
+		Timeout: 1 * time.Second,
```

Comment:
Also please lower this timeout. `go test` currently takes about 3.8s on my computer so we don't exactly want to add an extra second on top of that. You just have to find something non-flaky, but perhaps around 50-100ms.

Reply:
50ms seems to be reliable